pkg/services/system_intakes.go

```diff
@@ -109,11 +135,32 @@ func NewFetchSystemIntakeByID(
 			logger.Error("failed to fetch system intake")
 			return &models.SystemIntake{}, &apperrors.QueryError{
 				Err:       err,
-				Model:     "system intake",
+				Model:     "System intake",
 				Operation: apperrors.QueryFetch,
 			}
 		}
 		return intake, nil
 	}
 
 }
+
+// NewValidateAndSubmitSystemIntake is a service to submit system intake to CEDAR
+func NewValidateAndSubmitSystemIntake(
```

Comment:
What does exporting this as a service do rather than having an unexported function? You can pass the implementation right through the `NewSaveSystemIntake`

Reply:
nothing. good call out

Reply:
Oh, actually, it helps us mock out the submission in testing.

Reply:
Can you just mock what's in the `cedar` package instead though? That's where the actual implementation bits are. This is just calling that function.